.github/workflows/pr_bench.yml

```diff
@@ -0,0 +1,25 @@
+name: Benchmark pull requests
+on: [pull_request]
+
+env:
+  CARGO_TERM_COLOR: always
+
```

Comment:
Please add a statement to not keep this job running if the PR was updated, similar to:
https://github.com/lurk-lab/neptune/blob/4c1244dbeecaf8252cedaed25f0f7be24082ebfc/.github/workflows/rust.yml#L33-L35